docs/source/contributor_guide/installation_developer.rst

```diff
@@ -0,0 +1,53 @@
+Installation
+============
+
+Getting the source code
+-----------------------
+The source files for nbgrader and its documentation are hosted on GitHub. To
+clone the nbgrader repository::
+
+    git clone https://github.com/jupyter/nbgrader
+    cd nbgrader
+
+Installing flit and building nbgrader
+-------------------------------------
+**Note**: Python 3 is required for this step using flit.
+You will use ``pip3`` to install `flit <https://github.com/takluyver/flit>`_
+and then use ``flit`` to install and build nbgrader::
+
+    pip3 install flit
```

Comment:
You can also remove the bit about python 3, see the changes here: https://github.com/jupyter/nbgrader/pull/409/files#diff-6a3371457528722a734f3c51d9238c13L36